src/css/components/app-modal.less

```diff
@@ -0,0 +1,47 @@
+.app-modal {
+  .modal-content {
+    .app-error-block {
+      padding: @base-spacing-unit @base-spacing-unit*2.5;
+      background: fadeout(@text-color-danger-light, 75%);
+      &+.modal-body {
+        height: calc(~"80vh" - (@modal-header-height+@modal-footer-height+@modal-error-height));
+      }
+    }
+
+    .app-config-edit {
+      display: flex;
+      height: 100%;
+    }
+
+    .content-component {
+      border-left: @modal-footer-border-color 1px solid;
+      overflow: scroll;
+      padding-bottom: @base-spacing-unit*2;
+      padding-left: @base-spacing-unit*2;
+      padding-right: @base-spacing-unit*2.5;
+      padding-top: @base-spacing-unit*2;
+      width: 100%;
+    }
+
+    .modal-body {
+      height: calc(~"80vh" - (@modal-header-height+@modal-footer-height));
```

Comment:
As discussed either a fixed or a fluid height will be kinda hard to use in some edge cases. It’s the usual modal problem.